### DISCUSSION TITLE
Slideover for Select create option

### DISCUSSION THREAD

**Discussion by vincent-tarrit** [filamentphp/filament#5905] [Ideas]
Hello, i would love to use slideover for Select.creationOptionForm. Do you think that it's possible ? Guide me a bit and I would love to PR it !

Thank you !

**Comment by danharrin** [comment]
Try `createOptionAction(fn ($action) => $action->slideOver())`

**Reply by vincent-tarrit** [reply]
Thank you ! It works perfectly
